fuzzbench/ci_test.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
```

Comment:
Most of our tests start with the word "test_".
I don't really think this is a good place to change it so that they end with "_test".

Reply:
Done